addons/account/views/report_invoice.xml

```diff
@@ -156,6 +156,16 @@
                                             </t>
                                         </tr>
                                     </t>
+                                    <t t-if="o.amount_total == o.amount_untaxed">
+                                        <tr style="">
+                                            <td>
+                                                <span>Tax 0% on <span t-field="o.amount_untaxed"/></span>
```

Comment:
About the style I would use the same as when there is only one kind of tax in the invoice.

```suggestion
                                                <span>Tax 0%</span>
```